src/commands/createNewProject/NewProjectLanguageStep.ts

```diff
@@ -106,13 +108,17 @@ export class NewProjectLanguageStep extends AzureWizardPromptStep<IProjectWizard
 
         const wizardOptions: IWizardOptions<IProjectWizardContext> = { promptSteps, executeSteps };
 
-        // All languages except Java support creating a function after creating a project
+        const isPythonV2Plus = context.language === ProjectLanguage.Python && context.languageModel && context.languageModel > 1;
```

Comment:
Can we reuse the `isPythonV2Plus` utility?

Reply:
Consolidated